tests/e2e/pom/paper-view-model.ts

```diff
@@ -0,0 +1,34 @@
+import { expect, type Locator, type Page } from "@playwright/test";
+
+export class DevPaperViewPage {
+    readonly page: Page;
+    readonly allListEntries: Locator;
+    readonly referenceListEntry0: Locator;
+
+    constructor(page: Page) {
+        this.page = page;
+        this.allListEntries = page.getByRole("listitem");
+        this.referenceListEntry0 = page
```

Comment:
I think you wanted to use the `allListEntries` var, correct?

```suggestion
        this.referenceListEntry0 = allListEntries
```